src/data/real/cau_seq_filter.lean

```diff
@@ -57,127 +104,158 @@ let ⟨k, hk⟩ := nat.exists_eq_add_of_le hmn in
-         ... < ε/2 + ε/2 : add_lt_add hdist1 hdist2
-         ... = ε : add_halves _,
+  -- x : β,  hx : x ∈ t1 ∩ set_seq_of_cau_filter hf B hB N
+  -- we still have to show that x ∈ t2, i.e., edist x y < ε
+  have I1 : seq_of_cau_filter hf B hB N ∈ set_seq_of_cau_filter hf B hB n :=
+    (set_seq_of_cau_filter_monotone hf B hB (le_max_left n n2)) (seq_of_cau_filter_mem_set_seq hf B hB N),
+  have I2 : x ∈ set_seq_of_cau_filter hf B hB n :=
+    (set_seq_of_cau_filter_monotone hf B hB (le_max_left n n2)) hx.2,
+  have hdist1 : edist x (seq_of_cau_filter hf B hB N) < FB B hB n :=
+    set_seq_of_cau_filter_spec hf B hB _ I2 I1,
+  have hdist2 : edist (seq_of_cau_filter hf B hB N) y < ε / 2 :=
+    hn2 N (le_max_right _ _),
+  have hdist : edist x y < ε := calc
+    edist x y ≤ edist x (seq_of_cau_filter hf B hB N) + edist (seq_of_cau_filter hf B hB N) y : edist_triangle _ _ _
+          ... < FB B hB n + ε/2 : ennreal.add_lt_add hdist1 hdist2
+          ... ≤ ε/2 + ε/2 : add_le_add_right' (le_of_lt hnε)
+          ... = ε : ennreal.add_halves _,
   have hxt2 : x ∈ t2, from ht2' hdist,
-  existsi x,
-  apply ht1t2,
-  exact mem_inter hx.left hxt2
+  exact ne_empty_iff_exists_mem.2 ⟨x, ht1t2 (mem_inter hx.left hxt2)⟩
 end
 
 end
 end sequentially_complete
 
-/--A metric space in which every Cauchy sequence converges is complete-/
-theorem complete_of_cauchy_seq_tendsto {α : Type u} [metric_space α]
-  (H : ∀(u : ℕ → α), cauchy_seq u → ∃x, tendsto u at_top (nhds x)) :
+/-- An emetric space in which every Cauchy sequence converges is complete. -/
+theorem complete_of_cauchy_seq_tendsto {α : Type u} [emetric_space α]
+  (H : ∀u : ℕ → α, cauchy_seq u → ∃x, tendsto u at_top (nhds x)) :
   complete_space α :=
 ⟨begin
-  /-Consider a Cauchy filter `f`-/
+  -- Consider a Cauchy filter `f`
   intros f hf,
-  /-Introduce a sequence `u` approximating the filter `f`-/
-  let u := sequentially_complete.seq_of_cau_filter hf,
-  /-It is Cauchy-/
-  have : cauchy_seq u := sequentially_complete.seq_of_cau_filter_is_cauchy hf,
-  /-Therefore, it converges by assumption. Let `x` be its limit-/
+  -- Introduce a sequence `u` approximating the filter `f`. We don't need the bound `B`,
+  -- so take for instance `B n = 1` for all `n`.
+  let u := sequentially_complete.seq_of_cau_filter hf (λn, 1) (λn, ennreal.zero_lt_one),
+  -- It is Cauchy.
+  have : cauchy_seq u := sequentially_complete.seq_of_cau_filter_is_cauchy hf (λn, 1) (λn, ennreal.zero_lt_one),
+  -- Therefore, it converges by assumption. Let `x` be its limit.
   rcases H u this with ⟨x, hx⟩,
-  /-The original filter also converges to `x`-/
-  exact ⟨x, sequentially_complete.le_nhds_cau_filter_lim hf hx⟩
+  -- The original filter also converges to `x`.
+  exact ⟨x, sequentially_complete.le_nhds_cau_filter_lim hf (λn, 1) (λn, ennreal.zero_lt_one) hx⟩
 end⟩
 
+/- -A very useful criterion to show that a space is complete is to show that all sequences
```

Comment:
The docstring prefix is mixed up with white space:
```suggestion
/-- A very useful criterion to show that a space is complete is to show that all sequences
```

Reply:
Thanks